conf/global_params.config

```diff
@@ -28,6 +28,8 @@ cohort_tsv = "${params.project}_cohort.tsv"
 
 gatk38_jar = "${projectDir}/resources/GenomeAnalysisTK-3.8-0-ge9d806836/GenomeAnalysisTK.jar"
 
+gatk_jar_link = "https://storage.googleapis.com/gatk-software/package-archive/gatk/GenomeAnalysisTK-3.8-0-ge9d806836.tar.bz2"
+
```

Comment:
After some testing, preferably we can remove the necessity of `gatk38_jar` completely 🤔

Reply:
agreed, but we can keep it until we properly test it